website/static/js/pages/dashboard-page.js

```diff
@@ -24,13 +23,11 @@ var request = $.getJSON(url, function(response) {
     var uploadSelection = ko.utils.arrayFilter(allNodes, function(node) {
         return $.inArray(node.permissions, ['write', 'admin']) !== -1;
     });
-    // Filter out components and nodes for which user is not admin
-    var registrationSelection = ko.utils.arrayFilter(uploadSelection, function(node) {
-        return node.category === 'project' && node.permissions === 'admin';
-    });
+
+    var registrationSelection = uploadSelection;
```

Comment:
This doesn't appear to be correct. `uploadSelection` includes nodes for which the user has `write` permissions; `registrationSelection` should not.